src/server/active_tran_server.cpp

```diff
@@ -208,6 +210,28 @@ void active_tran_server::receive_log_page (cubpacking::unpacker &upk)
     }
 }
 
+void active_tran_server::receive_data_page (cubpacking::unpacker &upk)
+{
+  std::string message;
+  upk.unpack_string (message);
+
+  int error_code;
+  std::memcpy (&error_code, message.c_str (), sizeof (error_code));
+
+  if (prm_get_bool_value (PRM_ID_ER_LOG_READ_DATA_PAGE))
+    {
+      if (error_code == NO_ERROR)
+	{
+	  _er_log_debug (ARG_FILE_LINE,
+			 "Received data page message from Page Server."); // TODO: Ilie - print page info if no error.
```

Comment:
Unpack the page and print info.
```suggestion
	  char buf[IO_MAX_PAGE_SIZE];
	  std::memcpy (buf, message.c_str () + sizeof (error_code), db_io_page_size ());
	  const FILEIO_PAGE *io_page = (FILEIO_PAGE *) buf;
	  _er_log_debug (ARG_FILE_LINE,
			 "Received data page message from Page Server."); // TODO: Ilie - print io_page.prv.lsa & pageid & volid.
```